src/mscorlib/shared/System/MemoryExtensions.Fast.cs

```diff
@@ -359,6 +359,7 @@ public static bool StartsWith(this ReadOnlySpan<char> span, ReadOnlySpan<char> v
         /// <summary>
         /// Creates a new span over the portion of the target array.
         /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Did you mean to add it to this overload?  I ask because it looks relatively bulky and you didn't annotate the corresponding string/int AsSpan overload.

Reply:
This is trying to fix another non-nonsensical perf shard that we have today:

```
     char[];
     new Span<char>(a, 1 ,1); // Fastest: ForceInlined
     a.AsSpan(1,1); // Slower: ForceInlined, but the extra layer leaves collateral damage that the JIT is not able to get rid of 
     a.AsSpan(1); // Slowest: Not inlined
```

I agree that it is not obvious whether forceinlining this one is a good trade-off. We should be consistent at least - I will mark the string/int AsSpan overloads as well for consistency.